CharlieBackend.Business/Services/LessonService.cs

```diff
@@ -154,11 +156,16 @@ public async Task<LessonDto> UpdateLessonAsync(long id, UpdateLessonDto lessonMo
             }
         }
 
-        public async Task<IList<StudentLessonDto>> GetStudentLessonsAsync(long studentId)
+        public async Task<Result<IList<StudentLessonDto>>> GetStudentLessonsAsync(long studentId)
```

Comment:
Don't forget to call .ToActionResult in controller action. Otherwise, it will change schema of response body.